.travis.yml

```diff
@@ -30,8 +30,7 @@ jdk:
   - openjdk11
 branches:
   only:
-  - /.*/
-
+    - master
```

Comment:
Please, do a *full* rollback. I mean, leave the .travis.yml as it is in master branch. The file shouldn't appear in the diff tab of the PR. Or explain why the change is needed (maybe I'm missing something).

Reply:
Fixed in 05982ba4eefed043b201d8b95eb1c9b6190d2673